app/models/service_orchestration.rb

```diff
@@ -39,6 +38,10 @@ def update_orchestration_stack
 
   def orchestration_stack
     @orchestration_stack ||= service_resources.find { |sr| sr.resource.kind_of?(OrchestrationStack) }.try(:resource)
+    if @orchestration_stack.nil? && options.fetch_path(:orchestration_stack, 'type')
```

Comment:
Comment https://github.com/ManageIQ/manageiq/pull/11203#r79172437 answers this question.